packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -27,6 +27,8 @@ import {
     SummaryNode,
 } from "./summarizerNodeUtils";
 
+const defaultmaxUnreferencedTime = 7 * 24 * 60 * 60 * 1000; // 7 days
```

Comment:
please 'encode' the time unit in the name of the const (same as for the other variables and configs).